lib/metaData.js

```diff
@@ -4,11 +4,11 @@ const { compile: asyncapi } = require('@cap-js/asyncapi');
 const { COMPILER_TYPES } = require('./constants');
 const { Logger } = require('./logger');
 
-module.exports = async (data) => {
+module.exports = async (data, model = null) => {
     const parts = data?.split("/").pop().replace(/\.json$/, '').split(".");
     const compilerType = parts.pop();
```

Comment:
missing csn interop as compiler option @stewsk to give hint, how to trigger CAP compiler